samcli/local/apigw/service.py

```diff
@@ -354,7 +354,7 @@ def _construct_event(flask_request, port, binary_types):
 
         if is_base_64:
             LOG.debug("Incoming Request seems to be binary. Base64 encoding the request data before sending to Lambda.")
-            request_data = base64.b64encode(request_data)
+            request_data = base64.b64encode(request_data).decode()
         elif request_data:
```

Comment:
unify the decode in this elif block and the b64 statement

Reply:
done